test/unit-tests/components/views/dialogs/BaseDialog-test.tsx

```diff
@@ -21,6 +21,7 @@ describe("BaseDialog", () => {
         const dialog = container.querySelector('[role="dialog"]') as HTMLElement;
         dialog?.focus();
         await userEvent.keyboard("{Escape}");
+        await userEvent.keyboard("{Escape}");
```

Comment:
given the test is "calls onFinished when Escape is pressed", not "... when Escape is pressed *twice*", this looks wrong. At the very least it needs some comments to explain it.